src/m365/flow/commands/owner/owner-remove.spec.ts

```diff
@@ -0,0 +1,194 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { telemetry } from '../../../../telemetry';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+import { aadUser } from '../../../../utils/aadUser';
+import { aadGroup } from '../../../../utils/aadGroup';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Cli } from '../../../../cli/Cli';
+import { formatting } from '../../../../utils/formatting';
+const command: Command = require('./owner-remove');
+
+describe(commands.OWNER_REMOVE, () => {
+  const environmentName = 'Default-d87a7535-dd31-4437-bfe1-95340acd55c6';
+  const flowName = '1c6ee23a-a835-44bc-a4f5-462b658efc12';
+  const userId = 'bb46fcd8-2104-4f3d-982c-eb74677251c2';
+  const userName = 'john@contoso.com';
+  const groupId = '37a0264d-fea4-4e87-8e5e-e574ff878cf2';
+  const groupName = 'Test Group';
+  const requestUrlNoAdmin = `https://management.azure.com/providers/Microsoft.ProcessSimple/environments/${formatting.encodeQueryParameter(environmentName)}/flows/${formatting.encodeQueryParameter(flowName)}/modifyPermissions?api-version=2016-11-01`;
+  const requestUrlAdmin = `https://management.azure.com/providers/Microsoft.ProcessSimple/scopes/admin/environments/${formatting.encodeQueryParameter(environmentName)}/flows/${formatting.encodeQueryParameter(flowName)}/modifyPermissions?api-version=2016-11-01`;
+  const requestBodyUser = { 'delete': [{ 'id': userId }] };
+  const requestBodyGroup = { 'delete': [{ 'id': groupId }] };
+
+  let log: string[];
+  let logger: Logger;
+  let commandInfo: CommandInfo;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    auth.service.connected = true;
```

Comment:
Could you add the extra test stubs?
https://github.com/pnp/cli-microsoft365/pull/4545/files#diff-195628c809e6fb2923588c891e9fe2362a4422873efef5b27532a051cf9d065d